transmutagen/gensolve.py

```diff
@@ -217,14 +254,50 @@ def make_ijk(ij, N):
                     idx += 1
     return ijk
 
+
 def get_thetas_alphas(degree, prec=200, use_cache=True):
     rat_func = get_CRAM_from_cache(degree, prec, log=True, plot=False, use_cache=use_cache)
 
     thetas, alphas, alpha0 = thetas_alphas(rat_func, prec)
     return thetas, alphas, alpha0
 
+
+def pyne_decay_matrix(fromto):
+    if not HAVE_PYNE:
+        raise RuntimeError('Must have PyNE installed to generate decay matrix from '
+                           'PyNE data. Please either install PyNE or generate with '
+                           'decay_matrix_kind="none", or --decay-matrix=none from the '
+                           'command line.')
+    if pyne.utils.use_warnings():
+        pyne.utils.toggle_warnings()
```

Comment:
Does this disable them if they are enabled? I think we might already call this elsewhere.